scipy/stats/_hypotests.py

```diff
@@ -141,46 +141,59 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
-    distribution from descriptive statistics
+    Calculates the poisson mean test, the "e-test", for the mean difference of
+    two samples that follow a Poisson distribution from descriptive statistics.
+
+    This is a two-sided test for the null hypothesis that two independent
+    samples have identical average (expected) values.
+
+    Let :math:`X_{11},...,X_{1n_1}` and :math:`X_{21},...,X_{2n_2}` be
+    independent samples, from :math:`Poisson(\lambda_1)` and
+    :math:`Poisson(\lambda_2)` distributions. It is well known that :math:`X_1`
+    and :math:`X_2` are independent:
 
-    Let :math:`X_{11},...,X_{1n_1}` and :math:`X_{21},...,X_{2n_2}` be independent
-    samples respectively, from :math:`Poisson(\lambda_1)` and :math:`Poisson(\lambda_2)`
-    distributions. It is well known that
-    
     .. math:: X_1 = \sum_{i=1}^{n_1} X_{1i} \sim Poisson(n_1\lambda_1)
-    
-    independently of
-    
+
     .. math:: X_2 = \sum_{i=1}^{n_2} X_{2i} \sim Poisson(n_2\lambda_2)
-    
-    Let `count1` and `count1` be the observed values of :math:`X_1` and :math:`X_2`,
-    respectively. The problem of interest here is to test
-    
+
+    Let `count1` and `count1` be the observed values of :math:`X_1` and
+    :math:`X_2`, respectively. The problem of interest here is to test
+
     .. math::
        H_0: \lambda_1 = \lambda_2 + \mathtt{diff} \quad vs. \quad
        H_a: \lambda_1 \ne \lambda_2 + \mathtt{diff}
-    
-    for `two-sided` hypothesis, where :math:`\mathtt{diff} \ge 0` is a given
-    number, based on (`nobs1`, `count1`, `nobs2`, `count2`). `two-sided` and `greater`
-    cases are demonstrated by [1]_.The `less` hypothesis performed by switching the
-    arguments on `greater` hypothesis.
+
+    for right sided `greater` hypothesis where :math:`\mathtt{diff} \ge 0` is
+    a given number, based on (`nobs1`, `count1`, `nobs2`, `count2`). \
+    `two-sided` and `greater` cases are demonstrated by [1]_. The `less`
+    hypothesis performed by switching the arguments on `greater` hypothesis.
 
     Parameters
     ----------
-    count1, count2 : int
-        Count event of interest from the first and second samples respectively
-    nobs1, nobs2 : int
-        Sample size observed
-    diff : int or float, optional
+    count1 : int
+        Observed values of interest from sample 1.
+    nobs1:
+        Sample size from sample 1.
+    count2 : int
+        Observed values of interest from sample 2.
+    nobs2:
+        Sample size from sample 2.
+    diff : int of float, optional
         The difference of mean between two samples under null hypothesis
     alternative : {'two-sided', 'less', 'greater'}, optional
-        Which alternative hypothesis to test
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+          * 'two-sided'
+          * 'less': one-sided
+          * 'greater': one-sided
```

Comment:
We should have added descriptions for each of the alternatives. For examples, see https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ansari.html#scipy.stats.ansari or https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.mannwhitneyu.html.